install/kubernetes/cilium/README.md

```diff
@@ -119,7 +119,13 @@ contributors across the globe, there is almost always someone available to help.
 | encryption.secretName | string | `"cilium-ipsec-keys"` | Name of the Kubernetes secret containing the encryption keys. |
 | endpointHealthChecking.enabled | bool | `true` |  |
 | endpointRoutes.enabled | bool | `false` | Enable use of per endpoint routes instead of routing via the cilium_host interface. |
+| eni.EC2APIEndpoint | string | `""` | EC2 API endpoint to usee |
```

Comment:
My main reason was just because EC2 stands for `elastic compute cloud` -- I don't have any other reason, happy to do the change if you think this is important.

Reply:
It's worth doing a quick scan IMO to see if there's a precedent for capitalizing our Helm vars. The issue I'm thinking about is a user potentially not realizing the fact that this Helm var is capitalized and forgets to do it because they're likely pattern matching against all other Helm vars that aren't capitalized. Very minor case, but I know that I have done similar things in the past where I've guessed the Helm var based on pattern matching and turned out I was wrong :sweat_smile:

Reply:
@christarazi modified it to lowercase to keep in sync with the rest of options.